src/sections/Meshery/index.js

```diff
@@ -29,20 +30,24 @@ const MesheryPage = () => {
                 {/* Meshery is the cloud native manager. <br /> */}
                 Confidently design, deploy, and operate your infrastructure and workloads with Meshery.
               </p>
-              <Button primary className="banner-btn" title="How Meshery Works"
-                url="/cloud-native-management/meshery/operating-service-meshes">
-                <GiClockwork size={21} className="button-icon" />
-              </Button>
               <Button secondary className="banner-btn" title="Run Meshery"
                 url="/cloud-native-management/meshery/getting-started">
                 <FiDownloadCloud size={21} className="button-icon" />
               </Button>
+              <Button primary className="banner-btn" title="How Meshery Works"
+                url="/cloud-native-management/meshery/operating-service-meshes">
+                <GiClockwork size={21} className="button-icon" />
+              </Button>
             </Col>
-            <Col lg={6} md={6} className="meshery-hero-img">
+            <Col lg={6} md={6} className="meshery-hero-img desc-text">
               <video autoPlay muted loop controls className="meshery-video">
                 <source src={mesheryDemo} type="video/mp4" />
               </video>
               {/* <img className="meshery-sup-img" src={mesheryFullStack} alt="Meshery the multi-mesh manager" /> */}
+              <Button primary className="banner-btn" style={{ "margin-top": "17px", "background-color": "#EBC017" }}
```

Comment:
@tarunsamanta2k20 Please refer to the theme, when adding colors, and void explicit margin, use flex, grid, and padding... 
`margin-top` here seems fragile.

Reply:
Have you read the above comments? 
The requirements are the following:

1. The new button should open in a new tab.
2. After clicking on the new button, the button color shouldn't be changed.
3. The new button should be at the same level as the other two buttons.

What would be your suggestion other than this?

Reply:
If I remove the CSS it'll look like this.
![image](https://github.com/layer5io/layer5/assets/55488549/2adfd84e-37da-43f7-8ad2-99a530c274f5)

Reply:
If I apply padding from the top the the whole button width is getting increased.

Reply:
Let's discuss this in websites call. Add this as agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.69qskjv558bk) @tarunsamanta2k20